kumascript/macros/secureContext_inline.ejs

```diff
@@ -22,4 +22,4 @@ var str_tooltip = mdn.localString({
 });
 %>
 
-<span class="notecard inline secure" title="<%-str_tooltip%>"><%-str_title%></span>
+<span class="badge inline secure" title="<%-str_tooltip%>"><%-str_title%></span>
```

Comment:
A few nits:
- the variable `lang` is unused, we can get rid of it
- would be great if we add `user-select: none;` to the [`.badge` class](https://github.com/mdn/yari/blob/5d59c6b3e367e0c3b2058d0294d825a5066d588d/client/src/document/index.scss#L835)

Do we need [html escaped version](https://ejs.co/#docs) for translations?
```suggestion
<span class="badge inline secure" title="<%= str_tooltip %>"><%= str_title %></span>
```

Reply:
For html escape, read-only inline doesn't have it either.